eos/parameters/hadron-properties.yaml

```diff
@@ -365,32 +365,38 @@ groups:
           central: +1.519e-12
           min:     +1.512e-12
           max:     +1.526e-12
+          latex:    '$\tau_{B_d}$'
           comments: 'cf. [PDG2012], p. 84'
       'life_time::B_u' :
           central: +1.641e-12
           min:     +1.633e-12
           max:     +1.649e-12
+          latex:    '$\tau_{B_u}$'
           comments: 'cf. [PDG2012], p. 70'
       'life_time::B_s' :
           central: +1.516e-12
           min:     +1.505e-12
           max:     +1.527e-12
+          latex:    '$\tau_{B_s}$'
           comments: 'cf. [PDG-Live], September 2013'
       'life_time::B@Y(4S)' :
           central: +1.580e-12
           min:     +1.573e-12
           max:     +1.589e-12
+          latex:    '$\tau_{B(\Psi(4S))}$'
```

Comment:
Shouldn't this be a Upsilon instead of a Psi?

Reply:
ups